nuxeo-admin-console-web/angular-app/src/app/app.component.scss

```diff
@@ -0,0 +1,10 @@
+@import '~@hylandsoftware/hy-ui-icons/index.css';
```

Comment:
what is this css for ?

Reply:
To use css of hyland instead of `material ui` icon as mentioned in the doc.

> by default after adding `hyIcon` on mat-icon it becomes part of hyland icon but css is still part `material ui`.

https://ui.hyland.io/5932dab8d/v/41976/p/0716ad-icon/t/58d5e3